cmd/finch-daemon/router_utils_test.go

```diff
@@ -71,3 +73,69 @@ namespace = "test_namespace"
 	assert.Equal(t, "test_address", cfg.Address)
 	assert.Equal(t, "test_namespace", cfg.Namespace)
 }
+
+func TestCheckRegoFileValidity(t *testing.T) {
+	tests := []struct {
+		name          string
+		setupFunc     func() (string, func())
```

Comment:
These are run as subtests, so each `setupFunc` should take a `testing.T` as input to avoid asserting on the parent test.